### PR TITLE
Patch due to Cordova 2.0 API change

### PR DESCRIPTION
**Pull request by yyang**
Hi dmedvinsky,

I haven't checked the iOS folder yet but I believe it should have similar changes on API.   But this patch works for Cordova / Phonegap 2.0 + on Android now.

Can you please accept this pull request, or I shall also work out the iOS patch to make the JS call consistent and make a new request?

Best,
yyang

### CODE REVIEW

**Comment by dmedvinsky** [comment]
Hey yyang,

Ideally we should make a transition for both plugins simultaneously, so if you have an access to Mac machine and are able to provide the patch, I'll accept it at once. Otherwise I will probably be able to work on it this weekend as I'm currently pretty busy in a non-mobile project at work.

Thanks for the patch anyways!

**Comment by yyang** [comment]
Yes I'm able to provide patch, but please also wait until this weekend and I can **_test**_ it on my iPhone - I'm having some really troubling tasks with me now.

This is due to a change on the JavaScript library so it's really straight forward, but I really need to try it out.

File: Android/startapp.js [hunk 1]
```diff
@@ -12,6 +12,4 @@ StartApp.prototype.start = function(params, success, fail) {
     return cordova.exec(success, fail, 'StartApp', 'startApp', [component]);
 };
 
-cordova.addConstructor(function() {
-    cordova.addPlugin('startapp', new StartApp());
```
Comment (yyang) [inline Android/startapp.js:16]: Cordova no longer support `addPlugin`.

**Comment by dmedvinsky** [comment]
Sorry for the delay and thanks!

**Pull request merged**